app/models/region/RegionCompletionTable.scala

```diff
@@ -87,33 +83,29 @@ object RegionCompletionTable {
     * @return
     */
   def updateAuditedDistance(streetEdgeId: Int) = db.withTransaction { implicit session =>
-    val distToAdd: Float = streetEdgesWithoutDeleted.filter(_.streetEdgeId === streetEdgeId).groupBy(x => x).map(_._1.geom.transform(26918).length).list.head
-    val regionId: Int = streetEdgeNeighborhood.filter(_.streetEdgeId === streetEdgeId).groupBy(x => x).map(_._1.regionId).list.head
 
-    val q = for { regionCompletion <- regionCompletions if regionCompletion.regionId === regionId } yield regionCompletion
+    val distToAdd: Float = StreetEdgeTable.getStreetEdgeDistance(streetEdgeId)
```

Comment:
Replaced JTS.transform with that StreetEdgeTable func that does it in the query.